app/models/preqForProgram.py

```diff
@@ -2,7 +2,7 @@
 from app.models.program import Program
 from app.models.event import Event
 
-class ProgramEvent(baseModel):
+class PreqForProgram(baseModel):
```

Comment:
We're going to ditch the PreqForProgram table for now, and just put a field in the event table for it.  Talk to Sandesh about the changes if they have not been brought into development yet.   He has implemented them in a branch he was working on.